src/System.Net.Primitives/src/System/Net/NetworkInformation/NetworkInformationException.cs

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.ComponentModel;
+using System.Net.Sockets;
+using System.Runtime.InteropServices;
+
+namespace System.Net.NetworkInformation
+{
+    /// <devdoc>
+    ///    <para>
+    ///       Provides NetworkInformation exceptions to the application.
+    ///    </para>
+    /// </devdoc>
+    public class NetworkInformationException : Win32Exception
```

Comment:
Same question as in other networking libraries about the Win32Exception base type.